connection.go

```diff
@@ -281,14 +282,68 @@ func (c *connection) close(ctx context.Context) error {
 func (c *connection) login(ctx context.Context) error {
 	hasCompression := c.config.compression
 	c.config.compression = false
+
+	authRequest := AuthCommand{
+		UseCompression: false,
+		ClientName:     c.config.clientName,
+		DriverName:     fmt.Sprintf("exasol-driver-go %s", driverVersion),
+		ClientOs:       runtime.GOOS,
+		ClientVersion:  "(unknown version)",
+		ClientRuntime:  runtime.Version(),
+		Attributes: Attributes{
+			Autocommit:         boolToPtr(c.config.autocommit),
+			CurrentSchema:      c.config.schema,
+			CompressionEnabled: boolToPtr(hasCompression),
+		},
+	}
+
+	if c.config.accessToken != "" {
+		err := c.prepareLoginViaToken(ctx)
+		if err != nil {
+			return err
+		}
+		authRequest.AccessToken = c.config.accessToken
+	} else if c.config.refreshToken != "" {
+		err := c.prepareLoginViaToken(ctx)
+		if err != nil {
+			return err
+		}
+		authRequest.RefreshToken = c.config.refreshToken
+	} else {
+		password, err := c.prepareLoginViaPassword(ctx)
+		if err != nil {
+			return err
+		}
+		authRequest.Username = c.config.user
+		authRequest.Password = password
+	}
+
+	if osUser, err := user.Current(); err == nil && osUser != nil {
+		authRequest.ClientOsUsername = osUser.Username
+	} else {
+		logCouldNotGetOsUser(err)
+	}
+	log.Printf("Login via authRequest, %v", authRequest)
```

Comment:
Does this also log passwords and tokens?

Reply:
Is already removed